package.json

```diff
@@ -99,7 +99,7 @@
   },
   "dependencies": {
     "@across-protocol/across-token": "^1.0.0",
-    "@across-protocol/constants": "^3.1.5",
+    "@across-protocol/constants": "^3.1.7",
```

Comment:
This version isn't published yet, so CI currently fails.

Reply:
Fixed: https://github.com/across-protocol/sdk/pull/683/commits/6758639b9c8eaa05c300cc548f386393e7ae1ed4